inference/core/managers/active_learning.py

```diff
@@ -35,8 +35,8 @@ async def infer_from_request(
             model_id=model_id, request=request, **kwargs
         )
         active_learning_eligible = kwargs.get(ACTIVE_LEARNING_ELIGIBLE_PARAM, False)
-        active_learning_disabled_for_request = kwargs.get(
-            DISABLE_ACTIVE_LEARNING_PARAM, False
+        active_learning_disabled_for_request = getattr(
+            request, DISABLE_ACTIVE_LEARNING_PARAM, False
```

Comment:
This now would raise an exception if `DISABLE_ACTIVE_LEARNING_PARAM` is not found on the `request` right? Do we need to handle the exception here or is that value guaranteed to be on the request?

Reply:
no, third parameter gives a default - exception will not be raised for requests that do not support AL